source/refined-github.css

```diff
@@ -138,3 +138,22 @@ pr-branches
 	min-height: 0;
 	margin: -5px 0;
 }
+
+/* Align author name on PR sticky header (GitHub bug) */
+.sticky-content .author.css-truncate-target {
+	vertical-align: -4px;
+}
+
+/* Reposition committer's avatar on PR commit (GitHub bug) */
+.pull-request-tab-content .commit .AvatarStack {
+	display: inline-block;
+	margin-bottom: -4px;
+	margin-right: -4px;
+}
+
+/* Add missing border radius to files (GitHub bug) */
+.js-blob-wrapper,
+.file:not(.open) .file-header.file-header--expandable {
```

Comment:
This works for both folded and unfolded files.

Note GitHub use `6px` for `border-top-left-radius` and `border-top-right-radius`, but I figured out `5px` should be the correct one.

Reply:
I think each description should link to this issue, otherwise it’s hard to understand what’s being fixed without blaming the line multiple times.

Reply:
Maybe also add links for other styles like https://github.com/sindresorhus/refined-github/pull/3352?